server/server.js

```diff
@@ -65,6 +65,7 @@ function renderFullPage(html, preloadedState) {
       <meta http-equiv="cache-control" content="no-cache">
       <meta http-equiv="Expires" content="0">
       <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1"/>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.2.0/normalize.min.css"/>
```

Comment:
Actually, if this is going to cause issues with webpack. Don't worry. Just leave them external for now and we'll figure it out later.